tf2_eigen/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ include_directories(include
 
 catkin_package(
   INCLUDE_DIRS include
-  DEPENDS EIGEN3)
+  DEPENDS EIGEN3 tf2)
```

Comment:
Yes, that's undeclared. But to be correct I believe it should be `CATKIN_DEPENDS` http://wiki.ros.org/catkin/CMakeLists.txt there's a nice detailed description [here](https://answers.ros.org/question/58498/what-is-the-purpose-of-catkin_depends/?answer=58593#post-id-58593) 
```suggestion
  CATKIN_DEPENDS tf2
  DEPENDS EIGEN3)
```